docs/install-local-env.md

```diff
@@ -1,31 +1,184 @@
-## Setup SCT locally development environment
+# Setting up SCT with uv and direnv
 
-This is the recommend why of building a local development environment
-that isn't depended on the distro python version.
+This guide provides instructions for setting up the SCT (Scylla Cluster Tests) development environment using [uv](https://docs.astral.sh/uv/) as the Python package manager and [direnv](https://direnv.net/) for automatic environment management.
 
-To run SCT tests locally run following::
+## Benefits of this setup
+
+1. **Fast dependency resolution**: uv is significantly faster than pip
+2. **Automatic environment management**: direnv handles activation/deactivation
+3. **Reproducible environments**: Lock files ensure consistency
+4. **Isolation**: Each project has its own environment
+5. **Modern tooling**: Leverages the latest Python packaging standards
+6. **Cross-platform compatibility**: Works on Linux, macOS, and Windows (WSL)
+
+## Prerequisites
+
+Before you begin, ensure you have the following installed on your system:
+
+- Git
+- A supported operating system (Linux, macOS, or Windows with WSL)
+- Basic system development tools (build-essential, etc.)
```

Comment:
I think this is redundant, also `- A supported operating system (Linux, macOS, or Windows with WSL)` is werid requirement to have installed your system

Reply:
WSL is weird, 
but SCT won't work on anything else but linux (one might be lucky to get it working under docker in macOS)